# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libdvb/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libdvb-0.5.5.1.orig/sample_progs/cam_menu.hh    2008-01-28 10:23:53.000000000 +0100
+++ libdvb-0.5.5.1/sample_progs/cam_menu.hh         2008-01-28 10:22:55.000000000 +0100
@@ -2,6 +2,7 @@
  * cam_menu.hh
  *
  */
+#include <cstring>
 #include <sys/types.h>
 #include <sys/socket.h>

--- libdvb-0.5.5.1.orig/libdvbmpeg/ringbuffy.c	2007-03-16 17:53:31.000000000 +0100
+++ libdvb-0.5.5.1/libdvbmpeg/ringbuffy.c	2007-03-16 17:54:10.000000000 +0100
@@ -19,6 +19,7 @@
 */
 
 #include "ringbuffy.h"
+#include <string.h>
 
 int ring_init (ringbuffy *rbuf, int size)
 {
